tests/tnimfrompy.py

```diff
@@ -8,10 +9,29 @@
 greet_args = ("world", )
 greet_kwargs = {'greeting': "Hello", }
 assert(s.greet(*greet_args, **greet_kwargs) == "Hello, world!")
+
+try:
+    s.greet()
+except TypeError as e:
+    if sys.version_info[0] == 2:
+        assert("TypeError('greet() takes exactly 2 arguments (0 given)',)" in repr(e))
```

Comment:
Not sure if I did something wrong or it's a normal diff between py2 and py3

Reply:
I think you can just check `greet() takes exactly 2 arguments (0 given)` in `repr(e)` to avoid version branching. Also add `doAssert(false)` at the end of the `try` block to make sure the exception happens.